### PR TITLE
Coupons 2.0 changes

### PR DESCRIPTION
**Pull request by bhelx**
These are some of the changes for coupons. To be merged before these two:

* https://github.com/recurly/recurly-client-net/pull/100
* https://github.com/recurly/recurly-client-net/pull/101

Approvers: @shulmang

### CODE REVIEW

File: Library/CouponRedemption.cs [hunk 1]
```diff
@@ -9,6 +9,7 @@ namespace Recurly
     public class CouponRedemption : RecurlyEntity
     {
 
+        public string UUID { get; private set; }
```
Comment (bhelx) [inline Library/CouponRedemption.cs]: @shulmang Looking at this again with this lib fresh on my mind, what would you think about me changing this to `Uuid` to be consistent with the other instance variables?

Comment (shulmang) [inline Library/CouponRedemption.cs]: @bhelx maintaining consistency makes sense to me.

**Comment by shulmang** [comment]
:+1:

**Comment by bhelx** [comment]
Should be published in 1.2.4 https://www.nuget.org/packages/recurly-api-client/

**Pull request merged**